src/data/resume-data.tsx

```diff
@@ -18,8 +18,71 @@ import {
   YearProgressLogo,
 } from "@/images/logos";
 import { GitHubIcon, LinkedInIcon, XIcon } from "@/components/icons";
+import { StaticImageData } from "next/image";
+type ResumeData = {
```

Comment:
I think the resume data should not have a type otherwise is going to cause a lot of problems for people who just want to fork the project and use it. some users may not have a project, websiteurl or don't want to put out location links etc.

Reply:
Yup, Fair point